src/serialization.jl

```diff
@@ -109,7 +227,7 @@ function from_dict(
     end
 
     # if !get_runchecks(portfolio)
```

Comment:
Remove this comment of runchecks and open an issue for evaluating if we want to do checks on portfolios